doc/source/output/json.rst

```diff
@@ -149,16 +158,6 @@ gcovr/excluded: boolean
   in particular with :ref:`exclusion markers`.
   May be absent if false.
 
-gcovr/decision: object
-  The :ref:`decision <json_format_decision>` entry for this line, if any.
-  Absent if there is no decision to report.
-  Requires that :option:`--decisions` coverage analysis was enabled.
-
-gcovr/calls: object
-  The :ref:`call <json_format_call>` for this line, if any.
-  Absent if there is no call to report.
-  Requires that :option:`--calls` coverage analysis was enabled.
```

Comment:
You dropped this line when moving, I assume that was intentional?

Reply:
No. That was the thing fixed in #1090 and they where not removed anymore when the option isn't used.